src/full/Agda/TypeChecking/Serialise/Instances/Common.hs

```diff
@@ -378,10 +378,10 @@ instance EmbPrj A.ModuleName where
   value n           = A.MName `fmap` value n
 
 instance EmbPrj A.Name where
-  icod_ (A.Name a b c d e) = icodeMemo nameD nameC a $
-    icodeN' (\ a b -> A.Name a b . underlyingRange) a b (SerialisedRange c) d e
+  icod_ (A.Name a b c d e f) = icodeMemo nameD nameC a $
```

Comment:
Do you need tricks here akin to serializing the Fixity separately.  This was needed to cope with renaming fixities, do we need something similar for plain renaming now?

Reply:
I don't think so. I've just added a field for the original name which isn't affected by renaming. Everything else should be the same as before.